pennylane/ops/op_math/composite.py

```diff
@@ -156,9 +156,11 @@ def eigendecomposition(self):
                 eigenvectors of the operator.
         """
         if self.hash not in self._eigs:
-            Hmat = self.matrix()
-            Hmat = math.to_numpy(Hmat)
-            w, U = np.linalg.eigh(Hmat)
+            mat = (
+                self.matrix()
+            )  # we no longer check for hermiticity and allow "anything" on simulator
```

Comment:
maybe fix the placement of the comment to avoid the weird formatting?

Reply:
Removed this since we added a hermiticity check 👍🏼